incubator/jenkins-operator/values.yaml

```diff
@@ -0,0 +1,17 @@
+replicaCount: 1
+image:
+  repository: quay.io/samsung_cnct/jenkins-operator
+  tag: 0.1.7
+  pullPolicy: IfNotPresent
+nameOverride: ''
+fullnameOverride: ''
+keepCRDs: false
+rbac: true
+chartCrds: true
```

Comment:
In addition to the readme there are usually comments around properties in the values.yaml file. For example, see https://github.com/helm/charts/blob/master/stable/etcd-operator/values.yaml. Can you please add some comments around the non-typical properties.

Reply:
added